src/HomeWork/StudentGrade.java

```diff
@@ -0,0 +1,41 @@
+package HomeWork;
```

Comment:
create a package for this task too
and pay attention that package's names should be start from a lower-case letter and should contain 1 word)